format.c

```diff
@@ -382,7 +382,7 @@ fd_format_impl(char buf[restrict static 128], const FdInstr* instr, uint64_t add
             if (disp && (has_base || has_idx)) {
                 *buf++ = (int64_t) disp < 0 ? '-' : '+';
                 if ((int64_t) disp < 0)
-                    disp = -disp;
+                    disp = -(int64_t) disp;
```

Comment:
Nope. If `disp` is `INT64_MIN`, the signed negation (assuming two's complement) will be undefined behavior, but the unsigned negation is valid and correct.

Reply:
Ah, good point. Let's just silence this warning then!